.gitmodules

```diff
@@ -2,3 +2,6 @@
 	path = ethcore/res/ethereum/tests
 	url = https://github.com/ethereum/tests.git
 	branch = develop
+[submodule "nsis/win-build"]
```

Comment:
The plugin should be just installed on the build machine along with nsis, no need to keep it in the repo